test/browser/features/lib/url-generator.rb

```diff
@@ -5,7 +5,7 @@ def initialize(maze_uri, fixtures_uri)
   end
 
   def for_path(path)
-    uri = URI.join(@fixtures_uri, path)
+    uri = URI.join(@maze_uri, "/docs#{path}")
```

Comment:
Prefixing with docs here allows: `Given I navigate to the test URL "/vue-router"` 

However, the text fixture and assertions still need to be well aware they're actually being served from `/docs/vue-router`. So I wonder if it would be better not to do the prefix and instead require `Given I navigate to the test URL "/docs/vue-router"`. Potentially seems a bit clearer?

Reply:
I agree - including `/docs` in all steps would definitely be clearer. I would prefer to go for that approach.

Reply:
Ah yes, I see your perspective on this now.  At first, naively, `/docs` seemed more like an incidental detail of the test harness to me, but now I see why it matters in the context of browsers and URLs.